### DISCUSSION TITLE
Quad9 got removed?

### DISCUSSION THREAD

**Discussion by Jackie7610** [bromite/bromite#2033] [General]
Hi there. In the most recent version of Bromite (101.0.4951.39) I’ve noticed that Quad9 went missing from the ‘Use Secure DNS’ section in ‘Privacy and Security’. Has it been removed? If so...why?

Thank you.

**Comment by n-buna404** [comment]
> Has it been removed? If so...why?

Yes, but google remove it by upstream in latest 101 chromium. Bromite just followed the upstream

**Comment by M86xKC** [comment]
If you selected it before version 101.0.4951.39 you would still be using it.

It is just not in the list provided by chromium anymore. you can still set it to `https://dns.quad9.net/dns-query`

**Comment by uazo** [comment]
disabled [here](https://source.chromium.org/chromium/chromium/src/+/fc53736a978173a5d21948e8c4adb8741cc8138b)
